konduit-serving-core/src/main/java/ai/konduit/serving/pipeline/steps/InferenceExecutionerStepRunner.java

```diff
@@ -108,27 +108,13 @@ public InferenceExecutionerStepRunner(PipelineStep pipelineStep) {
     }
 
     @Override
-    public void destroy() {
+    public void close() {
         inferenceExecutioner.stop();
     }
 
     @Override
-    public Map<String, SchemaType[]> inputTypes() {
-        Map<String, SchemaType[]> ret = new LinkedHashMap<>();
-        for (int i = 0; i < pipelineStep.getInputNames().size(); i++) {
-            ret.put(pipelineStep.getInputNames().get(i), new SchemaType[]{SchemaType.NDArray});
-        }
-        return ret;
-    }
-
-    @Override
-    public Map<String, SchemaType[]> outputTypes() {
-        Map<String, SchemaType[]> ret = new LinkedHashMap<>();
-        for (int i = 0; i < pipelineStep.getOutputNames().size(); i++) {
-            ret.put(pipelineStep.getOutputNames().get(i), new SchemaType[]{SchemaType.NDArray});
-        }
-
-        return ret;
+    public PipelineStep<?> getPipelineStep() {
+        return pipelineStep;
     }
```

Comment:
Do we need this considering we have it implemented in the BaseStepRunner?

Reply:
No, we don't. I'll remove it.